defrag/__init__.py

```diff
@@ -53,7 +51,7 @@
 else:
     config = configparser.ConfigParser()
     config.read("config.ini")
```

Comment:
Just because this makes my life easier ;)
```suggestion
    config.read(["config.ini", "opensuse.ini", "redis.ini", "opengm.ini"])
```